sessions/js-inputs-and-strings/multiple-inputs/css/styles.css

```
@@ -7,16 +7,17 @@
 body {
   margin: 0;
   font-family: system-ui;
+  background-color: #ddd;
 }
 
 .form {
   display: grid;
   grid-gap: 20px;
```

the more modern property would be to use gap instead
```suggestion
  gap: 20px;
```